ajax/libs/imgareaselect/package.json

```diff
@@ -26,5 +26,13 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/odyniec/imgareaselect"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/odyniec/imgareaselect.git",
+    "basePath": "distfiles",
+    "files": [
+      "**!(*.txt)/*"
```

Comment:
But I used git auto-update to test it, then I didn't get the ```.txt``` files.
The ```.txt``` files are in the ```distfiles``` directory, not in the ```css``` and ```scripts``` directory.